src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -269,9 +269,27 @@ static bool filterConversion(Conversion conversion)
                     ensureAllUnderlyingConversionsChecked(syntax, source, conversion, wasCompilerGenerated, destination, diagnostics);
                 }
 
+                var boundSource = BindToNaturalType(source, diagnostics);
+
+                if (!hasErrors && !isCast && conversion.Kind == ConversionKind.ImplicitPointer &&
```

Comment:
>isCast [](http://example.com/codeflow?start=35&length=6)

It looks like we are not propagating ```isCast``` to ```ensureAllUnderlyingConversionsChecked```, therefore, the suppression is likely not working when tuple conversion is involved. Need to test that.

Reply:
> Need to test that.

Not sure how - function pointers cannot be in tuple since they cannot be type arguments. Similar nullable. Other one I see is dynamic which doesn't need `isCast`, though.

Reply:
But actually that doesn't matter if we are disallowing implicit function pointer conversions, we don't need warnings for them.